### PR TITLE
We should use Intents to call other Activities

### PR DESCRIPTION
**Pull request by mkpietrzyk**
This PR is to prevent crashes on Marshmallow while calling Activities outside of MainActivity.

### CODE REVIEW

File: android/src/main/java/com/robertsheao/RNZenDeskSupport/RNZenDeskSupportModule.java [hunk 1]
```diff
@@ -228,12 +231,25 @@ public void callSupport(ReadableMap customFields) {
 
     ZendeskConfig.INSTANCE.setCustomFields(fields);
 
-    ContactZendeskActivity.startActivity(getReactApplicationContext(), null);
+    Activity activity = getCurrentActivity();
+
+    if(activity != null){
```
Comment (RobertSheaO) [inline android/src/main/java/com/robertsheao/RNZenDeskSupport/RNZenDeskSupportModule.java]: Please update this to !==

File: android/src/main/java/com/robertsheao/RNZenDeskSupport/RNZenDeskSupportModule.java [hunk 2]
```diff
@@ -228,12 +231,25 @@ public void callSupport(ReadableMap customFields) {
...
+    }
   }
 
   @ReactMethod
   public void supportHistory() {
-    RequestActivity.startActivity(getReactApplicationContext(), null);
+
+    Activity activity = getCurrentActivity();
+
+    if(activity != null){
```
Comment (RobertSheaO) [inline android/src/main/java/com/robertsheao/RNZenDeskSupport/RNZenDeskSupportModule.java]: Please update this to !==

Comment (mkpietrzyk) [reply android/src/main/java/com/robertsheao/RNZenDeskSupport/RNZenDeskSupportModule.java]: Yep, just a sec :)

**Comment by mkpietrzyk** [comment]
I have ommited by quick copy-pasting to change some names. Added your fix to statement. Should be ok now :)

**Comment by mkpietrzyk** [comment]
@RobertSheaO Mind if you tell me how to do that? This is my first contribution ever, so I sound like a noob :) Just copy-paste the link / commit nr?

**Pull request merged**